pkgs/by-name/he/helix/package.nix

```diff
@@ -39,12 +41,28 @@ rustPlatform.buildRustPackage rec {
     cp contrib/helix.png $out/share/icons/hicolor/256x256/apps
   '';
 
-  meta = with lib; {
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  # `hx --version` outputs 25.1 instead of 25.01
+  preVersionCheck = ''
+    export version=${lib.replaceStrings [ "0" ] [ "" ] version}
```

Comment:
Probably fine for now, but wouldn't this also turn `25.10` into `25.1` for example?

Reply:
I think something like `lib.replaceStrings [ ".0" ] [ "." ] version` would work, but also can't think of something less ugly...

Reply:
Is this solved?

Reply:
I asked about the leading zero at helix, and it seems like it was due to rust/cargo. They "fixed" it upstream now, so the next release can be version-checked without the "hack".